pkg/service/service.go

```diff
@@ -794,7 +790,6 @@ func (s *Service) upsertServiceIntoLBMaps(svc *svcInfo, onlyLocalBackends bool,
 		SessionAffinityTimeoutSec: svc.sessionAffinityTimeoutSec,
 		CheckSourceRange:          checkLBSrcRange,
 		UseMaglev:                 svc.useMaglev(),
-		L7LBProxyPort:             svc.l7LBProxyPort,
```

Comment:
Why is this being removed when a previous commit added it? Can we simplify the commits instead?

Reply:
Yes, there is a lot to be cleaned with the commits that I did not have the time for just yet.